### PR TITLE
fix: migrate storybook to v.6.1.21 [ci visual]

### PR DESCRIPTION
**Pull request by droshev**
## Description
migrate storybook to v. 6.1.21

BREAKING CHANGE:
update role attributes for avatar, generic tile and popover

#### Please check whether the PR fulfills the following requirements

1. Testing
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
2. Documentation
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot [hunk 1]
```diff
@@ -434,29 +422,25 @@ exports[`Storyshots Components/Avatar Group Group Type 1`] = `
           
                     
           <span
-            aria-label="John Doe"
+            aria-label="John Doe></span>
```
Comment (InnaAtanasova) [inline stories/avatar-group/__snapshots__/avatar-group.stories.storyshot:425]: missing closing " for the label

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot [hunk 2]
```diff
@@ -434,29 +422,25 @@ exports[`Storyshots Components/Avatar Group Group Type 1`] = `
...
+                </div>    
+                <div class="
             class="fd-avatar fd-avatar--m fd-avatar--circle fd-avatar--border fd-avatar--thumbnail"
-            role="presentation"
+            fd-avatar-group__item"=""
```
Comment (InnaAtanasova) [inline stories/avatar-group/__snapshots__/avatar-group.stories.storyshot:429]: something's wrong here

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot [hunk 3]
```diff
@@ -253,29 +253,25 @@ exports[`Storyshots Components/Avatar Group Group Type 1`] = `
           
                     
           <span
-            aria-label="John Doe"
+            aria-label="John Doe></span>
```
Comment (InnaAtanasova) [inline stories/avatar-group/__snapshots__/avatar-group.stories.storyshot:256]: missing closing "

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot [hunk 4]
```diff
@@ -253,29 +253,25 @@ exports[`Storyshots Components/Avatar Group Group Type 1`] = `
...
+                </div>    
+                <div class="
             class="fd-avatar fd-avatar--m fd-avatar--circle fd-avatar--border fd-avatar--thumbnail"
-            role="presentation"
+            fd-avatar-group__item"=""
```
Comment (InnaAtanasova) [inline stories/avatar-group/__snapshots__/avatar-group.stories.storyshot:260]: something's wrong here with the " "

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot [hunk 5]
```diff
@@ -308,29 +304,25 @@ exports[`Storyshots Components/Avatar Group Group Type 1`] = `
           
                     
           <span
-            aria-label="John Doe"
+            aria-label="John Doe></span>
```
Comment (InnaAtanasova) [inline stories/avatar-group/__snapshots__/avatar-group.stories.storyshot:307]: same as above

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot [hunk 6]
```diff
@@ -308,29 +304,25 @@ exports[`Storyshots Components/Avatar Group Group Type 1`] = `
...
+                </div>    
+                <div class="
             class="fd-avatar fd-avatar--m fd-avatar--circle fd-avatar--border fd-avatar--thumbnail"
-            role="presentation"
+            fd-avatar-group__item"=""
```
Comment (InnaAtanasova) [inline stories/avatar-group/__snapshots__/avatar-group.stories.storyshot:311]: same as above

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot [hunk 7]
```diff
@@ -363,29 +355,25 @@ exports[`Storyshots Components/Avatar Group Group Type 1`] = `
           
                     
           <span
-            aria-label="John Doe"
+            aria-label="John Doe></span>
```
Comment (InnaAtanasova) [inline stories/avatar-group/__snapshots__/avatar-group.stories.storyshot:358]: same as above

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot [hunk 8]
```diff
@@ -363,29 +355,25 @@ exports[`Storyshots Components/Avatar Group Group Type 1`] = `
...
+                </div>    
+                <div class="
             class="fd-avatar fd-avatar--m fd-avatar--circle fd-avatar--border fd-avatar--thumbnail"
-            role="presentation"
+            fd-avatar-group__item"=""
```
Comment (InnaAtanasova) [inline stories/avatar-group/__snapshots__/avatar-group.stories.storyshot:362]: same as above

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot (hunk 1 above)
Comment (droshev) [reply stories/avatar-group/__snapshots__/avatar-group.stories.storyshot]: corrected

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot (hunk 2 above)
Comment (droshev) [reply stories/avatar-group/__snapshots__/avatar-group.stories.storyshot]: corrected

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot (hunk 3 above)
Comment (droshev) [reply stories/avatar-group/__snapshots__/avatar-group.stories.storyshot]: corrected

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot (hunk 4 above)
Comment (droshev) [reply stories/avatar-group/__snapshots__/avatar-group.stories.storyshot]: corrected

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot (hunk 5 above)
Comment (droshev) [reply stories/avatar-group/__snapshots__/avatar-group.stories.storyshot]: corrected

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot (hunk 6 above)
Comment (droshev) [reply stories/avatar-group/__snapshots__/avatar-group.stories.storyshot]: corrected

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot (hunk 7 above)
Comment (droshev) [reply stories/avatar-group/__snapshots__/avatar-group.stories.storyshot]: corrected

File: stories/avatar-group/__snapshots__/avatar-group.stories.storyshot (hunk 8 above)
Comment (droshev) [reply stories/avatar-group/__snapshots__/avatar-group.stories.storyshot]: corrected

**Review by InnaAtanasova — APPROVED** [review]
Please rebase main and fix the conflicts

**Pull request merged**